packages/sdk/src/router/v6/addresses.ts

```diff
@@ -38,3 +38,8 @@ export const ZeroExV4Module: ChainIdToAddress = {
 };
 
 export const ZoraModule: ChainIdToAddress = {};
+
+export const BlurModule: ChainIdToAddress = {
+  [Network.Ethereum]: "0x0000000000000000000000000000000000000000",
```

Comment:
If the contracts are not yet deployed can simply leave the mapping empty rather than pointing to the zero address.